app/controllers/users_controller.rb

```diff
@@ -157,7 +157,7 @@ def handle_printable_instructions
       flash[:password] = @user.password
       redirect_to(action: :login_instructions, id: @user.id)
     else
-      redirect_to(index_url_with_page_num)
+      redirect_to(index_url_with_context)
```

Comment:
Howz about a quick happy path feature spec? remember_context_spec.rb:

1. Create N+2 responses where N is the per_page for responses. All submitted by the same user.
1. Go to response index
1. search by submitter:Your Fake user
1. click page 2
1. delete first response (don't forget to click confirm)
1. should be redirected back to page 2 with search intact